tests/unit/test_evt_tag_mem.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2022 OneIdentity
```

Comment:
Adding a space (One Identity) will make the copyright-check pass (but an LGPL or 3-Clause BSD test would be better IMHO, especially under lib/eventlog).